### ISSUE TITLE
Saving test images in main

### ISSUE THREAD

**Issue by ghost** [slinshady89/dynamic_distortion_calibrator#9]
Hi all,

yet another issue, where I'd like some input. :)

Here's the thing:
Since cv::imshow doesn't work, I'm trying to work around this. I want to achieve this by making openframeworks save the created images (as well as later the undistorted image). Unfortunately it doesn't work.
If I try to save the image, it simply doesn't work. If I don't run the `findRawDistortion` function beforehand, it does work. However if I then try to run the `findRawDistortion` function, the `findRawDistortion` function breaks. These should not be influencing each other in any way and I have no clue why they do.

Does anyone of you have any idea? Something that I could try out?

EDIT: Simply creating the test image once and then only running the `findRawDistortion` function won't work, because we also need to store the undistorted image (and for that we need the distortion maps generated by `findRawDistortion`).

Cheers!

**Comment by slinshady89** [comment]
Hey does it maybe fail because we write -1 into the mat objects? Maybe there should be a zero or so for the ofImage types?

**Comment by ghost** [comment]
No, we're not storing the object initialised with -1. We're storing a completly new image, caputred in the `ImageCreator`, it should have absolutelty zero to do with anything we're doing before or after. I'll try it out anyway, just maybe that does influence each other in some super obscure of way...
For some reason the `save` function cannot find the file  type (.jpg) if we run `findRawDistortion`, but can find the file type without problem if we don't.

**Comment by slinshady89** [comment]
I'm home from work now. That was just an idea i got at the trainstation this morning. I will look on it now and do some research
